Onedrive backups guide (#2)
worldbank/dime-standards: dime-research-standards/pillar-4-data-security/data-security-resources/onedrive-backup-guidelines.md

# DIME Analytics - Data Security Guides - WB OneDrive Back-up

This guide is for the World Bank's Enterprise version of OneDrive (WB OneDrive),

> and not all parts of this guide should be followed using regular OneDrive, DropBox or any other synching service.

Reviewer: ```suggestion
and not all parts of this guide should be followed using regular OneDrive, DropBox or any other syncing service.
```

> Those services are not secure enough for us to recommend that you upload identified data

Reviewer: ```suggestion
The non WB OneDrive services are not secure enough for us to recommend for uploading identified data
```

> without manually encrypt it first.

Reviewer: ```suggestion
without manually encrypting it first.
```

A folder created in WB OneDrive cannot be synced to a non-World Bank device

> even if that person has been invited to the folder.

Reviewer: ```suggestion
even if owner of that device has been invited to the folder.
```

This is because it is installed with extra layers of encryption that is connected to our WB computer log-ons.
While that makes it an unsuitable tool for day-to-day collaboration with team members that do not have a World Bank computer,
it makes it an excellent tool for back-ups of raw data,

> where there is no risk that data is lost because encryption keys are lost.

Reviewer: ```suggestion
where there is no risk of losing data as a result of loss of encryption keys.
```

## WB OneDrive back-up folders

> There is nothing special you have to do to set up a back-up folder in WB OneDrive, but since data back-up is such an important step, we still write a guide for it. The only way the back-up folder should differ from more most WB OneDrive folders is that this folder should not be synched to any device, and it should not be shared with anyone.

Reviewer: ```suggestion
While an easy process, setting up a back-up folder in WB OneDrive is an extremely important process.  The only way the back-up folder should differ from more most WB OneDrive folders is that this folder should not be synced to any device, and it should not be shared with anyone.
```

continuous (meaning we keep getting new data) or
discrete (meaning that once we have the data set it will not be updated again).

An example of a discrete source of data is a typical survey round.
Once we are done collecting data we will not get more data for that survey round.
That data should be backed up in WB OneDrive after the data collection is completed
but before the data collection server is closed down.
For continuous data sources there is no equally obvious point when the data should be backed up,
but it should be often enough that a potential loss of data between back-ups
is not too devastating for the project.

<img src="https://github.com/worldbank/dime-standards/blob/master/dime-research-standards/pillar-4-data-security/data-security-resources/img/onedrive-backup-3.png" width="75%"><!--- Image is read from master branch or use full URL-->

Once you have created the folder for your data source,
simply drag all the files you want to back up here.
You can drag a whole folder and all its content at the same time.
You should back up the most raw version of your data, even if that is a bunch of csv files.

> Since storage space will not be an issue since you will not sync this folder,

Reviewer: ```suggestion
Since storage space will not be an issue as you will not sync this folder,
```

> you can back up both the rawest files and a data set where all of them are merged.

Reviewer: ```suggestion
you can back up both, the rawest files and a data set created from the raw files.
```
